x/pki/x509/x509_test.go

```diff
@@ -51,6 +51,25 @@ func Test_DecodeCertificates(t *testing.T) {
 	require.Equal(t, testconstants.RootSerialNumber, certificate.SerialNumber)
 	require.Equal(t, testconstants.RootSubject, certificate.Subject)
 	require.Equal(t, testconstants.RootSubjectKeyID, certificate.SubjectKeyID)
+
+	// decode root google certificate with vid
```

Comment:
I would create a separate test method for decoding certificates with VID/PID.

Reply:
Done! ✅